server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -266,7 +267,7 @@ public void verifyReposThenStopServices() {
             final AtomicBoolean cleanedUp = new AtomicBoolean(false);
             continueOrDie(cleanupResponse, r -> cleanedUp.set(true));
 
-            runUntil(cleanedUp::get, TimeUnit.MINUTES.toMillis(1L));
+            runUntil(cleanedUp::get, TimeUnit.MINUTES.toMillis(5L));
```

Comment:
I'm investigating why this is taking so long, until then upping the value here to because even though I found a seed that fails here now I bet there's one where this could fail in one of the other master failover tests.